### PR TITLE
XWIKI-20781: Livedata colors are hard coded

### PR DESCRIPTION
**Pull request by Sereza7**
**Jira:** https://jira.xwiki.org/browse/XWIKI-20781

**PR Changes:**
* Replaced hard coded values with color theme values, trying to match the closest: value-wise (considering the Iceberg color theme) and then semantically-wise.
* Replaced some semi-transparent elements with full colors (from the theme), some uses of transparency seemed superfluous

**View:**
20781ViewAnnotated

**Note:**
 Those changes will only change code quality and not accessibility of the interface as long as https://jira.xwiki.org/browse/XWIKI-20680 is not closed.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -715,7 +715,7 @@
       <dependency>
         <groupId>io.sf.carte</groupId>
         <artifactId>css4j</artifactId>
-        <version>3.9.1</version>
+        <version>4.0</version>
```
Comment (manuelleduc) [inline pom.xml:718]: I guess this is not part of the PR?

Comment (Sereza7) [reply pom.xml:718]: When I tried to build (to take screenshots and update), I had an issue, and did a fetch - rebase - push force with lease.
However the two last commits in the history of the PR as of now are still here...

**Comment by Sereza7** [comment]
**View:**
Theme Name | Before |  After
:---------:|:-------------------------:|:-------------------------:
Iceberg | 20781-IcebergBefore | 20781-IcebergAfter
Darkly | 20781-DarklyBefore | 20781-DarklyAfter
Garden | 20781-GardenBefore | 20781-GardenAfter
Kitty | 20781-KittyBefore | 20781-KittyAfter

As we can see on these examples, this PR does not change radically the style of LiveData.
The slight variations are "smart" in that they follow the theme conventions: for Iceberg, Garden and Kitty, we can see that the links on the right of each entry ( Edit & Delete ) are a bit darker after the fix, while they are a bit lighter on Darkly, which is a dark theme. In those 4 cases, it fixes contrasts issues on the table.
